core/src/main/java/com/netflix/conductor/core/events/ActionProcessor.java

```diff
@@ -48,158 +48,145 @@
 @Singleton
 public class ActionProcessor {
 
-	private static Logger logger = LoggerFactory.getLogger(ActionProcessor.class);
-
-	private WorkflowExecutor executor;
-
-	private MetadataService metadata;
-	
-	private static final ObjectMapper om = new ObjectMapper();
-	
-	private ParametersUtils pu = new ParametersUtils();
-	
-	@Inject
-	public ActionProcessor(WorkflowExecutor executor, MetadataService metadata) {
-		this.executor = executor;
-		this.metadata = metadata;
-	}
-
-	public Map<String, Object> execute(Action action, String payload, String event, String messageId) throws Exception {
-
-		logger.debug("Executing {}", action.getAction());
-		Object jsonObj = om.readValue(payload, Object.class);
-		if(action.isExpandInlineJSON()) {
-			jsonObj = expand(jsonObj);
-		}
```

Comment:
Why are above 4 lines removed?

Reply:
that was a mistake, i added the inline json payload expansion back